src/azure-cli/azure/cli/command_modules/reservations/_params.py

```diff
@@ -31,3 +31,17 @@ def load_arguments(self, _):
 
     with self.argument_context('reservations catalog show') as c:
         c.argument('reserved_resource_type', arg_type=get_enum_type(ReservedResourceType))
+
+    with self.argument_context('reservations reservation-order calculate') as c:
+        c.argument('billing_scope_id', options_list=['--scope'])
+        c.argument('reserved_resource_properties', options_list=['--reserved-resource-properties'])
+        c.argument('renew', arg_type=get_three_state_flag())
```

Comment:
Here you are using get_three_state_fls() for your paramater, in this way what will happen if setting --renew false in your command?

Reply:
renew should have a default value of false. Setting this to true will automatically purchase a new reservation on the expiration date time.

Reply:
Because when you use get_three_state_flags() here, help message will show up with "Allowed values: false, true". What if use `--renew false` in your command? 
Another question is for expiration date time for the reservation to be purchased. Can you tell what is the expiration date time?

Reply:
No, we cannot make the decision for the customer. This is an optional value. If the customer did not pass any value or use --renew false. We will not renew the reservation. 
As part of the purchase responese, expiration data will only display after the purchase succeeded. 
cause we need to make sure we will not charge the customer before they purchased Reservation.